### DISCUSSION TITLE
Add `alignment_mode` parameter to Span.char_span()

### DISCUSSION THREAD

**Discussion by itssimon** [explosion/spaCy#11983] [New Features & Project Ideas]
`Doc.char_span()` has the `alignment_mode` parameter, but `Span.char_span()` does not. As a user I would expect both of these functions to be consistent.

**Comment by adrianeboyd** [comment]
No particular reason. I suspect users who wanted this probably just switched to `Doc.char_span`. Would you like to submit a PR?

**Reply by itssimon** [reply]
Sure, happy to give it a go!
